### PR TITLE
Instructions for GPU Sept preview installation

### PR DESCRIPTION
**Pull request by baziwane**

### CODE REVIEW

File: preview/GPU/gpu.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+- Some manual configuration steps are needed to configure the Linux workernodes once the target cluster is set up.
+- GPU-enabled node pools running with `Standard_NK12` have not been thoroughly tested. We recommend using `Standard_NK6` only.
+- If you allocate more worker nodes than available GPUs, this causes a VM leak (the VM appearing in off state). This VM has no impact on the cluster and should be cleaned up by either running remove-akshcicluster or uninstall-akshci. This issue will be resolved in an upcoming release. 
+- On a cluster with 4 GPUs, if you start off with 1 worker node enabled with GPU, then scale up to 4 nodes, the set-akshcinodepool command incorrectly reports that cluster does not have enough resources however, all the worker nodes are properly created.
+- **This preview requires a clean install, upgrading GPU-enabled node pools has not yet been thoroughly tested**
```
Comment (pradipd) [inline preview/GPU/gpu.md:26]: As mentioned in the teams thread, let's update this: 
```
[9:45 AM] Madhan Raj Mookkandy
Let the customer decide here.
If they want to do fresh install, let them do it.
If they want to do upgrade, please suggest the right steps 
```

File: preview/GPU/gpu.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+NVIDIA Software. The software may include components developed and owned by NVIDIA Corporation or its licensors. The use of these components is governed by the NVIDIA end user license agreement located at https://www.nvidia.com/content/DriverDownload-March2009/licence.php?lang=us.
+
+## Known issues and limitations
+
+- VMs with GPU enabled are not added to HA clustering in Windows Server 2019, Windows Server 2002 or Azure Stack HCI. This functionality will be available in a later version of Windows Server and Azure Stack HCI.
+- There is a 1:1 mapping of GPU to VM.
+- GPU enabled VMs are not pinned to a specific worker node and will not automatically failover to another physical node. AKS-HCI will recreate the VM on another physical node should the node hosting the current VM go down. This might incur application downtime during this preview if the application is not redundantly setup.
+- Some manual configuration steps are needed to configure the Linux workernodes once the target cluster is set up.
```
Comment (pradipd) [inline preview/GPU/gpu.md:22]: This is no longer true.  there should be no manual configuration!

File: preview/GPU/gpu.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+It should just work
+
+### What happens if I attempt to create a GPU-enabled node pool or scale a node pool but all physical GPUs are already assigned?
+
+When you create a GPU enabled or scale a GPU enabled node pool but all physical GPUs are already assigned and there are no resources available, you will see an error such as: `Error: The Host does not have enough hardware (GPU) resources to complete the <add|Set>-AksHciNodePool request. Make sure there are enough resources available and try again.` 
+
+Recommendation: <TO DO>
+
+### What happens if I have physical GPUs installed but AKS cluster can't detect the device?
```
Comment (pradipd) [inline preview/GPU/gpu.md:253]: Oh. Let's delete this section. this is old.

Comment (pradipd) [reply preview/GPU/gpu.md:253]: Unfortunately, i don't have new recommendations.

**Review by pradipd — APPROVED** [review]

**Comment by baziwane** [comment]
#signoff

**Pull request merged**